lib/cellular/models/sms.rb

```diff
@@ -41,16 +40,6 @@ def delivered?
       @delivered
     end
 
-    def country
-      warn "[DEPRECATION] 'country' is deprecated; use 'country_code' instead"
-      @country_code
-    end
-
-    def country=(country)
-      warn "[DEPRECATION] 'country' is deprecated; use 'country_code' instead"
-      @country_code = country
-    end
-
```

Comment:
So I'm not entirely sure if this belongs in this PR. But now that it's here, I'm sure we can get away with it. :wink: